http/http.go

```diff
@@ -184,6 +185,13 @@ func (h *HTTP) RunServer() (err error) {
 
 	log.Infof("Service listening on %s", GetConfig().ListenAddress)
 
+	// Since main blocks here until completion, tell systemd we're ready.
+	// This is a no-op if NOTIFY_SOCKET isn't set.
+	if os.Getenv("NOTIFY_SOCKET") != "" {
+		log.Debug("Notifying systemd of readiness")
+	}
+	systemd.SdNotify(false, systemd.SdNotifyReady)
```

Comment:
Why not include the `SdNotify` call inside the condition too?

Reply:
Yeah, I guess I was just trying to avoid skipping the actual `SdNotify` implementation, but I guess it does the same check. I'll amend.